ext_localconf.php

```diff
@@ -2,12 +2,19 @@
 defined('TYPO3_MODE') or die();
 
 $initialize = function ($extKey) {
+    $isVersion9Up = \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 9000000;
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTypoScriptSetup(
         '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/TypoScript/Extbase/setup.txt">'
     );
-    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
-        '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/PageTSconfig/Suggest.tsconfig">'
-    );
+    if ($isVersion9Up) {
+        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+            '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:' . $extKey . '/Configuration/PageTSconfig/Suggest_9.tsconfig">'
```

Comment:
I would suggest to go the other way round and use `Suggest_prior_9.tsconfig` and leave `Suggest.tsconfig` as the file `Suggest.tsconfig` will last longer in this extension. What do you think?

Reply:
Hi Manuel, that sounds good, doing it the other way, to support newer versions by default and the older ones with a special part.